js/dialogcards.js

```diff
@@ -529,6 +535,24 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
     $cardText.html(text);
     $cardText.toggleClass('hide', (!text || !text.length));
   };
+  
+  /**
+   * Change image of card, used when turning cards.
+   *
+   * @param $card
+   * @param imgSrc
+   */
+  C.prototype.changeImage = function ($card, imgSrc) {
```

Comment:
Image is changed momentarily once we click the "turn" button, image should change when the card has transitioned halfway through.